sdk/storage/storage-blob/README.md

```diff
@@ -233,7 +233,7 @@ const { BlobServiceClient } = require("@azure/storage-blob");
 const account = "<account name>";
 const sas = "<service Shared Access Signature Token>";
 
-const blobServiceClient = new BlobServiceClient(`https://${account}.blob.core.windows.net${sas}`);
+const blobServiceClient = new BlobServiceClient(`https://${account}.blob.core.windows.net/${sas}`);
```

Comment:
SAS token should be directly appended to the URI, there shouldn't be a '/' between base URI and SAS token.

Reply:
@EmmaZhu If you believe the changes in this PR are not required, then place close the PR